### DISCUSSION TITLE
Ajouter un hyperlien dans un formulaire

### DISCUSSION THREAD

**Discussion by billo04** [MTESSDev/FRW#58] [Annonces]
Est-ce possible de faire afficher un hyperlien dans un formulaire FRW?

Exemple:
L'hyperlien ouvrant une nouvelle page dans laquelle l'utilisateur trouvera de l'information supplémentaire.
![image](https://user-images.githubusercontent.com/124705516/231847197-a6cfdeae-17c6-425d-80eb-8fb275f66822.png)

**Comment by j0ber** [comment]
Bonjour,

C'est possible de mettre le HTML que vous désirez (donc évidemment un hyperlien).

Voir  [cet exemple](https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=sectionStyle) (voir exemple 3).

On vient de l'ajouter.
